src/providers/aws.rs

```diff
@@ -556,7 +556,7 @@ where
 
                 let res =
                     futures_util::future::join_all(self.regions.drain().map(|(region, mut rl)| {
-                        let region_span = tracing::debug_span!("region", %region);
+                        let region_span = tracing::debug_span!("region", region = %region);
```

Comment:
You shouldn't need the `region = ` here. `tracing` uses the name of the variable as the field name by default.